### PR TITLE
Update Repo With CANIDs from excel sheet

### PR DESCRIPTION
**Pull request by younesr1**
This PR was made in response to the comments left on Sahil's PR, which has been merged in.

### CODE REVIEW

File: config/hw_bridge.h [hunk 1]
```diff
@@ -4,27 +4,57 @@
 namespace ROVERCONFIG {
 constexpr uint16_t ROVER_MOTOR_PWM_FREQ_HZ = 20000;   // 20 kHz
 constexpr uint32_t ROVER_CANBUS_FREQUENCY  = 500000;  // 500 kbps
+constexpr uint16_t ROVER_CANID_FILTER_MASK = 0xFE0;   // Use bits [5:10] for addressing and 0:7 for command/message type
+}  // namespace ROVERCONFIG
+
+namespace CANFILTER {
 enum filtering_t {
```
Comment (jetkov) [inline config/hw_bridge.h:11]: Some of these are still incorrect, I made a typo for arm (the real first RX ID is 0x720). TX and RX are also wrong for some (take a look at the spreadsheet columns D & E

https://docs.google.com/spreadsheets/d/1-q8Ut22iqccHrx9dzZxglOa9t4iAxWaMrEmVZOrzj-8/edit?usp=sharing

Comment (jetkov) [reply config/hw_bridge.h:11]: Not necessary but I'd probably include them in the canID_t enum in their corresponding places so it is clear to someone which section they are modifying, etc.

**Review by jetkov — CHANGES_REQUESTED** [review]

File: config/hw_bridge.h [hunk 2]
```diff
@@ -4,27 +4,57 @@
...
+  SCIENCE,
+  GIMBTONOMY,
+  ROBOTEQ = 0x701
+};
+}  // namespace CANHEARTBEAT
+
+namespace CANID {
+enum canID_t {
+  // DRIVETRAIN CAN IDs
+  TPDO1 = 0x181,
```
Comment (wmmc88) [inline config/hw_bridge.h:47]: @akeaveny could u modify the drivetrain code to use these structs once these prs are in

File: config/hw_bridge.h (hunk 1 above)
Comment (younesr1) [reply config/hw_bridge.h:11]: apologies for late response.. i think ill leave it as it is. i wanted to differentiate between filters, heartbeats, errors, and regular canids

**Review by jetkov — APPROVED** [review]

**Pull request merged**